### ISSUE TITLE
Chat announcement

### ISSUE THREAD

**Issue by Nemiix1** [bytzo/sessility#8]
Is it possible to change the colour and font of the text that announces when a player goes sessile and when they become motile, Its the same colour as when a player leaves and joins and I wish to be able to make it grey and italic.

**Comment by bytzo** [comment]
[Formatting codes](https://minecraft.wiki/w/Formatting_codes) are supported if you want to change the color and style of motility messages. However, there is currently a bug causing translatable arguments (the `%s` that represents a player's name) to reset the message formatting, making it impossible to format the player name along with the rest of the motility message.

<p><img src="https://github.com/bytzo/sessility/assets/69266322/4621c46d-7da0-405b-9b58-ef284f46a287" width="720px"></p>

I have pushed a fix for this bug, which for now can be downloaded under the Artifacts section of [this page](https://github.com/bytzo/sessility/actions/runs/8042358268) until a proper release of Sessility is out. Below, I've also provided an example snippet of a `sessility.properties` file that uses formatting codes to make the motility messages grey and italic.

```properties
message-sessile=§7§o%s became a sessile creature
message-motile=§7§o%s returned from the depths
```

And here is the updated build, in action! (Ignore the polar bear, I was fine):

<p><img src="https://github.com/bytzo/sessility/assets/69266322/67a511c5-fce6-47db-913a-cd2754cc2dc6" width="720px"></p>

Let me know if this build works as expected for you!

**Comment by bytzo** [comment]
This issue should be resolved with the release of Sessility 0.6! It should now be possible to have almost complete control over the formatting of sessile and motile messages.
